### PR TITLE
Change "revenue metrics" wording to "cash sales metrics"

### PR DESCRIPTION
**Pull request by razvanpapadopol**
Part of 775-gh-Automattic/experimentation-platform
See 775-gh-Automattic/experimentation-platform#issuecomment-1258335299 for details.

* Update wording for:
  - Metrics directory
  - Experiment wizard (metric assignment)
  - Min-diff calculator
  - Creating new metrics
  - Experiment results

* Capitalize parameter name in Metrics Directory
Screen Shot 2022-10-01 at 01 34 48
Screen Shot 2022-10-01 at 01 31 54

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)

### CODE REVIEW

File: src/components/experiments/wizard/MinDiffCalculator.tsx [hunk 1]
```diff
@@ -199,16 +200,26 @@ const MinDiffCalculator = ({
...
-              <StepLabel>What is the baseline monthly revenue?</StepLabel>
+            <Step active key='cash-sales-2'>
+              <StepLabel>
+                What is the baseline monthly{' '}
+                <Link
+                  underline='always'
+                  href='https://fieldguide.automattic.com/the-experimentation-platform/experiment-metrics/#what-is-the-difference-between-cash-sales-and-revenue-metrics'
+                  target='_blank'
+                >
+                  Cash Sales
```
Comment (aaronyan) [inline src/components/experiments/wizard/MinDiffCalculator.tsx:211]: I'm thinking we lowercase this to "cash sales" in all the places, what do you think?

File: src/components/experiments/wizard/__snapshots__/ExperimentForm.test.tsx.snap [hunk 2]
```diff
@@ -1813,7 +1813,7 @@ exports[`sections should be browsable by the section buttons and show validation
                                   An Attribution Window
                                 </a>
                                  
-                                is the window of time after exposure to an experiment that we capture metric events for a participant (exposure can be from either assignment or specified exposure events). The refund window is the window of time after a purchase event. Revenue metrics will automatically deduct transactions that have been refunded within the metric’s refund window.
+                                is the window of time after exposure to an experiment that we capture metric events for a participant (exposure can be from either assignment or specified exposure events). The refund window is the window of time after a purchase event. Cash Sales metrics will automatically deduct transactions that have been refunded within the metric’s refund window.
```
Comment (aaronyan) [inline src/components/experiments/wizard/__snapshots__/ExperimentForm.test.tsx.snap:1816]: Lowercase to "Cash sales" here, if we do so.

**Review by aaronyan — COMMENTED** [review]
LGTM! Thanks so much for adjusting cash sales in all the places. What do you think about lowercasing all of the "Cash Sales" to "Cash sales" for the start of a sentence and "cash sales" if in the middle of a sentence? We do this with "conversions" in all the places.

**Comment by razvanpapadopol** [comment]
> LGTM! Thanks so much for adjusting cash sales in all the places. What do you think about lowercasing all of the "Cash Sales" to "Cash sales" for the start of a sentence and "cash sales" if in the middle of a sentence? We do this with "conversions" in all the places.

Yes, it makes sense to use the normal letter case when in sentences and even in small texts like hints/placeholders. Done in https://github.com/Automattic/abacus/pull/784/commits/d562a3e607c59605d4306e169943fede6308c003 and the readability is much improved ✨ 

At the same time, I'd say we keep the capitalized format (see screenshots in the description above) when displaying it as the name of the metric. I've seen `Cash Sales` in old FG pages with business metrics and also in the [ExPlat - Metrics page](PCYsg-Fqe-p2#what-is-the-difference-between-cash-sales-and-revenue-metrics).

**Comment by aaronyan** [comment]
Very nice, looks and reads great. Thanks for making those changes, Razvan!

> At the same time, I'd say we keep the capitalized format when displaying it as the name of the metric

Sure, that looks good to me too!

**Review by aaronyan — APPROVED** [review]
LGTM! Thanks for adding changing all the copy from 'revenue' -> 'cash sales'.

**Pull request merged**